### DISCUSSION TITLE
Plan Community Presence at Kubecon NA 2023

### DISCUSSION THREAD

**Discussion by jberkus** [etcd-io/etcd#16186] [General]
We need to figure out what we'll be doing as a community for Kubecon Chicago.  I've started [a doc here](https://docs.google.com/document/d/1Xqg2be4jbGynwqf8hEnKv9YnNiirhBAe5xoPo4G2nVM/edit?usp=sharing), to lay things out; this discussion is for reference in the Etcd project, as well as for any additional comments.  

If you're available to help Etcd with Kubecon, please add your name (and your ideas) to the doc.

**Comment by wenjiaswe** [comment]
Thanks @jberkus ! I will be there and am more than happy to help with anything needed. I added myself to the doc.
